utils/lshw.go

```diff
@@ -348,6 +350,14 @@ func (l *Lshw) xNIC(node *LshwNode) *common.NIC {
 		BusInfo:     node.Businfo,
 	}
 
+	// Parse out PCI Vendor and Product identifiers from node.ProductName if available
+	pciIdRegex, _ := regexp.Compile(` \[(\S{4}):(\S{4})\]$`)
+	if matches := pciIdRegex.FindStringSubmatch(nic.ProductName); matches != nil {
+		nic.ProductName = pciIdRegex.ReplaceAllString(nic.ProductName, "")
+		nic.Common.PCIVendorID = matches[1]
+		nic.Common.PCIProductID = matches[2]
```

Comment:
A len() check here would be ideal, since this could panic if `len(matches) == 1`